OptimizelySDK/OptimizelyTests/OptimizelyTests-DataModel/ConditionLeaf.swift

```diff
@@ -0,0 +1,23 @@
+/****************************************************************************
```

Comment:
why is this class here?

Reply:
Good catch. Looks like an error introduced while merging. Will clean up xcode project.